bcm27xx: add support for Pi4 and switch bcm27xx from ext4 to squashfs

T-X opened the pull request:
- [ ] ~~Must be flashable from vendor firmware:~~ N/A, no vendor firmware
  - [ ] Web interface
  - [ ] TFTP
  - [x] Other: unpack image to microSD 
- [x] Must support upgrade mechanism
  - [x] Must have working sysupgrade
    - [x] Must keep/forget configuration (`sysupgrade [-n]`, `firstboot`)
  - [x] Gluon profile name matches autoupdater image name
        (`lua -e 'print(require("platform_info").get_image_name())'`)
- [ ] Reset/WPS/... button must return device into config mode
  - no buttons available (however with physical access the SD card can be updated on another device to reenable the config mode)
- [ ] Primary MAC address should match address on device label (or packaging)
      (https://gluon.readthedocs.io/en/latest/dev/hardware.html#hardware-support-in-
  - no lable available, but MAC seems fixed/stable upon `firstboot -y` (here: dc:a6:32:0c:6f:25)
packages)
  - When re-adding a device that was supported by an earlier version of Gluon, a
    factory reset must be performed before checking the primary MAC address, as
    the setting from the old version is not reset otherwise.
- Wired network
  - [x] should support all network ports on the device
  - [ ] must have correct port assignment (WAN/LAN): N/A, single port
    - if there are multiple ports but no WAN port:
      - the PoE input should be WAN, all other ports LAN
      - otherwise the first port should be declared as WAN, all other ports LAN
- Wireless network (if applicable)
  - [x] Association with AP must be possible on all radios
  - [ ] Association with 802.11s mesh must work on all radios: 11s/mesh radio unsupported by driver
  - [ ] AP+mesh mode must work in parallel on all radios: 11s/mesh radio unsupported by driver
- LED mapping
  - Power/system LED
    - [ ] Lit while the device is on: unstable, sometimes not lit
    - [ ] Should display config mode blink sequence: not working, is either persistenly lit or persistently not lit after booting into config-mode
          (https://gluon.readthedocs.io/en/latest/features/configmode.html)
  - ~~Radio LEDs:~~ no radio LED available (secondary, green LED is mapped to SD card activity)
    - [ ] Should map to their respective radio
    - [ ] Should show activity
  - Switch port LEDs
    - [ ] Should map to their respective port (or switch, if only one led present) 
    - [ ] Should show link state and activity
- ~~Outdoor devices only:~~ not an outdoor device
  - [ ] Added board name to `is_outdoor_device` function in `package/gluon-core/luasrc/usr/lib/lua/gluon/platform.lua`
- ~~Cellular devices only:~~ not a cellular device
  - [ ] Added board name to `is_cellular_device` function in `package/gluon-core/luasrc/usr/lib/lua/gluon/platform.lua`
  - [ ] Added board name with modem setup function `setup_ncm_qmi` to `package/gluon-core/luasrc/lib/gluon/upgrade/250-cellular`
- Docs:
  - [ ] Added Device to `docs/user/supported_devices.rst`

T-X commented:
Currently a Pi4 is planned to be used as WiFi rooftop "core node" at Freifunk Lübeck, to integrate 5GHz devices with vanilla OpenWrt in WDS-bridge mode and 60GHz devices with a Mikrotik RouterOS / vendor firmware: https://wiki.luebeck.freifunk.net/docs/infrastruktur/jls/#optimierungen-mit-vlans

T-X commented:
And one more note / small issue: Network hardware checksumming is not working properly. Something between br-client and the ethernet interface seems to mess it up again. Still seems to work ok, should only be a performance penalty.

```
[   40.559022] br-client: hw csum failure
[   40.562818] skb len=112 headroom=286 headlen=112 tailroom=1778
[   40.562818] mac=(232,14) net=(246,40) trans=286
[   40.562818] shinfo(txflags=0 nr_frags=0 gso(size=0 type=0 segs=0))
[   40.562818] csum(0xffff864f ip_summed=2 complete_sw=0 valid=0 level=0)
[   40.562818] hash(0x0 sw=0 l4=0) proto=0x86dd pkttype=2 iif=14
[   40.591624] dev name=br-client feat=0x00002007ffdd78e9
[   40.596773] skb headroom: 00000000: c1 00 00 b0 42 54 3b 91 21 64 39 91 24 00 80 52
[   40.604427] skb headroom: 00000010: e0 03 14 aa 69 ff ff 97 e0 03 13 aa b9 e0 ff 97
[   40.612077] skb headroom: 00000020: 00 00 80 52 f3 53 41 a9 f5 5b 42 a9 f7 1b 40 f9
[   40.619727] skb headroom: 00000030: fd 7b c5 a8 c0 03 5f d6 80 aa 44 b9 bf 02 00 71
[   40.627377] skb headroom: 00000040: a0 7f 4e 01 80 41 00 00 72 20 00 00 00 00 00 00
[   40.635027] skb headroom: 00000050: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
[   40.642676] skb headroom: 00000060: 01 80 2c 31 26 7a 00 00 00 80 0b 4c 8a 5e 00 00
[   40.650326] skb headroom: 00000070: 00 00 6a 58 02 06 00 00 00 01 04 00 01 02 00 00
[   40.657976] skb headroom: 00000080: 00 00 33 33 00 00 33 33 00 00 01 5c f2 a4 1c 55
[   40.665625] skb headroom: 00000090: 6c dc 86 dd 60 00 00 00 00 d2 11 01 fe 80 00 00
[   40.673275] skb headroom: 000000a0: 00 00 00 00 f0 a4 1c ff fe 55 6c dc ff 02 00 00
[   40.680925] skb headroom: 000000b0: 00 00 00 00 00 00 00 00 00 00 01 5c e0 56 12 b5
[   40.688574] skb headroom: 000000c0: 00 d2 00 00 08 00 00 00 d2 c3 9d 00 ff ff ff ff
[   40.696224] skb headroom: 000000d0: ff ff f2 a4 1c 55 6c dc 43 05 01 0f 2d 00 1d 5f
[   40.703873] skb headroom: 000000e0: bc d2 de ad ca fe aa cc 33 33 00 00 00 01 de ad
[   40.711523] skb headroom: 000000f0: ca fe aa aa 86 dd 60 01 7e a0 00 70 3a ff fe 80
[   40.719173] skb headroom: 00000100: 00 00 00 00 00 00 dc ad ca ff fe fe aa aa ff 02
[   40.726823] skb headroom: 00000110: 00 00 00 00 00 00 00 00 00 00 00 00 00 01
[   40.733954] skb linear:   00000000: 86 00 4e f8 40 00 03 84 00 00 00 00 00 00 00 00
[   40.741609] skb linear:   00000010: 03 04 40 c0 00 01 51 80 00 00 38 40 00 00 00 00
[   40.749261] skb linear:   00000020: fd ef ff c0 3d d7 00 00 00 00 00 00 00 00 00 00
[   40.756914] skb linear:   00000030: 03 04 40 c0 00 01 51 80 00 00 38 40 00 00 00 00
[   40.764566] skb linear:   00000040: 20 01 06 7c 2d 50 00 00 00 00 00 00 00 00 00 00
[   40.772215] skb linear:   00000050: 19 03 00 00 00 00 00 c8 20 01 06 7c 2d 50 00 00
[   40.779865] skb linear:   00000060: 00 00 00 00 00 00 aa aa 01 01 de ad ca fe aa aa
[   40.787514] skb tailroom: 00000000: 00 00 00 00 06 02 00 04 07 00 00 00 01 01 00 08
[   40.795164] skb tailroom: 00000010: 00 00 01 f4 00 00 00 fa 02 01 00 00 00 0f 2e 00
[   40.802814] skb tailroom: 00000020: 63 3f 9b ac de ad ca fe dd 01 6a c0 b7 b0 ed 69
[   40.810463] skb tailroom: 00000030: 00 be 00 30 04 01 00 14 01 02 00 02 fb fe 90 6c
[   40.818113] skb tailroom: 00000040: 80 01 00 00 11 09 04 16 00 00 00 00 02 01 00 00
[   40.825762] skb tailroom: 00000050: 06 02 00 04 07 00 00 00 01 01 00 08 00 00 01 f4
[   40.833412] skb tailroom: 00000060: 00 00 00 fa 00 0f 2d 00 94 20 48 fe ea 60 c0 90
[   40.841062] skb tailroom: 00000070: 60 3b 3a 01 f1 95 fb cd 00 bf 00 2c 04 01 00 1c
[   40.848711] skb tailroom: 00000080: 01 5e 00 03 85 8f a1 5d 80 01 00 00 cc b3 dc 7a
[   40.856361] skb tailroom: 00000090: 80 00 00 00 15 6f a7 28 00 00 00 00 06 02 00 04
[   40.864011] skb tailroom: 000000a0: 01 00 00 00 02 01 00 00 00 0f 2d 00 24 ba 21 a2
[   40.871660] skb tailroom: 000000b0: 32 17 f5 43 a5 1b 3a 01 f1 95 fb cd 00 9e 00 2c
[   40.879310] skb tailroom: 000000c0: 04 01 00 1c 01 33 00 03 2e f8 f2 20 80 01 00 00
[   40.886960] skb tailroom: 000000d0: 67 c4 8f 07 80 00 00 00 7b 0b 74 b2 00 00 00 00
[   40.894609] skb tailroom: 000000e0: 06 02 00 04 01 00 00 00 02 01 00 00 00 91 f5 5b
[   40.902259] skb tailroom: 000000f0: 02 a9 f5 03 01 aa f6 03 00 aa 21 00 80 52 e0 03
[   40.909909] skb tailroom: 00000100: 15 aa f3 53 01 a9 f7 1b 00 f9 46 de ff 97 1f 1c
[   40.917558] skb tailroom: 00000110: 00 72 40 06 00 54 e0 03 15 aa d7 02 14 91 46 f5
[   40.925208] skb tailroom: 00000120: ff 97 f3 03 00 aa a1 1a 00 91 e0 03 17 aa 02 00
[   40.932858] skb tailroom: 00000130: 80 d2 73 7e 40 92 25 43 00 94 f4 03 00 aa 00 02
[   40.940507] skb tailroom: 00000140: 00 b4 03 e0 00 91 e0 03 03 aa 3b f5 ff 97 1f 00
[   40.948157] skb tailroom: 00000150: 13 eb c1 00 00 54 e2 03 13 aa e1 03 15 aa e0 03
[   40.955806] skb tailroom: 00000160: 03 aa 8c dd ff 97 e0 02 00 34 e1 03 14 aa e0 03
[   40.963456] skb tailroom: 00000170: 17 aa 04 de ff 97 e0 03 14 aa fe df ff 97 61 e2
[   40.971106] skb tailroom: 00000180: 00 91 20 00 80 d2 67 df ff 97 f4 03 00 aa a0 02
[   40.978755] skb tailroom: 00000190: 00 b4 e2 03 13 aa e1 03 15 aa 00 e0 00 91 55 e0
[   40.986405] skb tailroom: 000001a0: ff 97 80 fa 00 91 80 16 00 f9 e1 03 14 aa e0 03
[   40.994055] skb tailroom: 000001b0: 17 aa 70 dd ff 97 c0 aa 40 b9 00 00 1d 32 c0 aa
[   41.001704] skb tailroom: 000001c0: 00 b9 00 00 80 52 f3 53 41 a9 f5 5b 42 a9 f7 1b
[   41.009354] skb tailroom: 000001d0: 40 f9 fd 7b c4 a8 c0 03 5f d6 40 00 80 52 fa ff
[   41.017004] skb tailroom: 000001e0: ff 17 20 01 80 52 f8 ff ff 17 fd 7b bd a9 fd 03
[   41.024653] skb tailroom: 000001f0: 00 91 f5 5b 02 a9 f5 03 00 aa f3 53 01 a9 f3 03
[   41.032303] skb tailroom: 00000200: 00 aa f4 03 01 2a a0 0e 4b f8 16 00 40 f9 1f 00
[   41.039952] skb tailroom: 00000210: 15 eb c1 01 00 54 a1 01 00 90 21 20 00 91 35 00
[   41.047602] skb tailroom: 00000220: 06 91 20 c0 40 f9 16 00 40 f9 1f 00 15 eb c1 01
[   41.055252] skb tailroom: 00000230: 00 54 9f 06 00 71 60 02 00 54 9f 7a 1e 72 21 03
[   41.062901] skb tailroom: 00000240: 00 54 01 00 80 d2 12 00 00 14 03 0c 40 f9 e2 03
[   41.070551] skb tailroom: 00000250: 14 2a e1 03 13 aa 60 00 3f d6 e0 03 16 aa d6 02
[   41.078201] skb tailroom: 00000260: 40 f9 eb ff ff 17 03 0c 40 f9 e2 03 14 2a e1 03
[   41.085851] skb tailroom: 00000270: 13 aa 60 00 3f d6 e0 03 16 aa d6 02 40 f9 eb ff
[   41.093500] skb tailroom: 00000280: ff 17 e0 03 13 aa 9b eb ff 97 61 c2 40 f9 f5 5b
[   41.101150] skb tailroom: 00000290: 42 a9 60 2e 40 f9 f3 53 41 a9 fd 7b c3 a8 61 1e
[   41.108799] skb tailroom: 000002a0: 00 14 f3 53 41 a9 f5 5b 42 a9 fd 7b c3 a8 c0 03
[   41.116449] skb tailroom: 000002b0: 5f d6 01 a0 40 b9 3f 0c 00 71 c0 03 00 54 fd 7b
[   41.124098] skb tailroom: 000002c0: be a9 62 00 80 52 fd 03 00 91 f3 53 01 a9 f3 03
[   41.131748] skb tailroom: 000002d0: 00 aa 1f 54 02 39 02 a0 00 b9 81 02 00 34 21 04
[   41.139398] skb tailroom: 000002e0: 00 51 3f 04 00 71 68 00 00 54 01 00 80 52 bf ff
[   41.147048] skb tailroom: 000002f0: ff 97 60 a2 0b 91 01 00 80 52 58 ed ff 97 74 a2
[   41.154697] skb tailroom: 00000300: 06 91 01 00 80 52 e0 03 14 aa 54 ed ff 97 e0 03
[   41.162347] skb tailroom: 00000310: 14 aa 94 f0 ff 97 e0 03 13 aa d9 e4 ff 97 f3 53
[   41.169996] skb tailroom: 00000320: 41 a9 fd 7b c2 a8 b7 43 00 14 41 00 80 52 f0 ff
[   41.177646] skb tailroom: 00000330: ff 17 c0 03 5f d6 fd 7b ae a9 fd 03 00 91 f3 53
[   41.185296] skb tailroom: 00000340: 01 a9 f4 03 00 aa f5 5b 02 a9 56 1c 00 12 d5 00
[   41.192945] skb tailroom: 00000350: 00 90 f7 63 03 a9 f7 03 01 aa b5 62 2f 91 f9 6b
[   41.200595] skb tailroom: 00000360: 04 a9 f0 df ff 97 03 00 80 d2 02 04 00 91 e1 43
[   41.208258] skb tailroom: 00000370: 01 91 00 bb 80 d2 07 e0 ff 97 e1 03 14 aa f3 03
[   41.215912] skb tailroom: 00000380: 00 aa 18 00 10 91 19 60 13 91 14 a0 0b 91 e0 2b
[   41.223565] skb tailroom: 00000390: 40 f9 c8 dd ff 97 60 2e 00 f9 60 c2 13 91 00 03
[   41.231217] skb tailroom: 000003a0: 0f a9 60 c2 02 91 60 02 0b a9 60 02 01 91 e1 03
[   41.238876] skb tailroom: 000003b0: 13 aa 60 02 04 a9 60 a2 06 91 19 e7 0d a9 16 e2
[   41.246542] skb tailroom: 000003c0: ff 97 e1 03 13 aa e0 03 14 aa 13 e2 ff 97 60 a2
[   41.254204] skb tailroom: 000003d0: 10 91 c2 ff ff b0 c1 ff ff b0 42 70 36 91 21 30
[   41.261866] skb tailroom: 000003e0: 1d 91 e8 dd ff 97 60 02 14 91 03 00 80 d2 02 00
[   41.269528] skb tailroom: 000003f0: 80 52 c1 ff ff 90 21 00 1e 91 16 de ff 97 00 00
[   41.277191] skb tailroom: 00000400: 00 f0 00 60 01 91 60 8e 00 f9 00 00 00 f0 00 90
[   41.284853] skb tailroom: 00000410: 08 91 60 c6 00 f9 c0 ff ff b0 00 f0 2c 91 60 aa
[   41.292515] skb tailroom: 00000420: 00 f9 e0 03 17 aa 7f c2 0b 39 e3 12 00 91 e2 63
[   41.300177] skb tailroom: 00000430: 01 91 21 03 80 52 bb f2 ff 97 e4 03 00 2a a0 02
[   41.307839] skb tailroom: 00000440: 24 91 c4 df ff 97 e0 3f 40 f9 7f 36 00 f9 80 00
[   41.315502] skb tailroom: 00000450: 00 b4 4e fa ff 97 b3 dd ff 97 60 36 00 f9 e0 37
[   41.323164] skb tailroom: 00000460: 40 f9 60 00 00 b5 75 ce 00 f9 16 00 00 14 a2 f4
[   41.330826] skb tailroom: 00000470: ff 97 f7 03 00 aa 80 ff ff b4 a1 00 00 f0 21 0c
[   41.338488] skb tailroom: 00000480: 22 91 30 dd ff 97 00 14 00 34 a0 01 00 90 00 00
[   41.346150] skb tailroom: 00000490: 3b 91 00 e0 0c 91 01 10 40 f9 a1 12 00 b5 75 ce
[   41.353812] skb tailroom: 000004a0: 00 f9 a2 00 00 f0 c1 00 00 b0 42 5c 3c 91 21 64
[   41.361475] skb tailroom: 000004b0: 39 91 e0 03 13 aa 04 00 80 52 03 00 80 d2 db fd
[   41.369137] skb tailroom: 000004c0: ff 97 60 ce 40 f9 03 38 40 b9 e0 3b 40 f9 00 12
[   41.376799] skb tailroom: 000004d0: 00 b4 6f f6 ff 97 00 1c 00 12 60 36 02 39 e0 8b
[   41.384461] skb tailroom: 000004e0: 40 f9 a0 11 00 b5 60 10 44 d3 60 4a 02 39 e0 53
[   41.392123] skb tailroom: 000004f0: 40 f9 76 4e 02 39 60 11 00 b4 65 f6 ff 97 00 1c
[   41.399785] skb tailroom: 00000500: 00 12 00 00 00 52 60 c6 06 39 e0 57 40 f9 e0 10
[   41.407447] skb tailroom: 00000510: 00 b4 5f f6 ff 97 00 1c 00 12 e1 5b 40 f9 00 00
[   41.415110] skb tailroom: 00000520: 00 52 60 ca 06 39 61 00 00 b4 e0 03 14 aa a4 f5
[   41.422772] skb tailroom: 00000530: ff 97 e1 5f 40 f9 61 00 00 b4 e0 03 14 aa 02 f6
[   41.430434] skb tailroom: 00000540: ff 97 e0 63 40 f9 60 00 00 b4 5e f8 ff 97 60 ae
[   41.438096] skb tailroom: 00000550: 04 b9 e0 67 40 f9 60 00 00 b4 5a f8 ff 97 60 aa
[   41.445757] skb tailroom: 00000560: 04 b9 e0 6f 40 f9 60 00 00 b4 56 f8 ff 97 60 32
[   41.453425] skb tailroom: 00000570: 13 39 94 01 00 f0 94 42 1c 91 77 f2 12 91 7f ba
[   41.461091] skb tailroom: 00000580: 04 b9 80 06 40 a9 e0 06 00 a9 20 00 80 52 60 2e
[   41.468754] skb tailroom: 00000590: 13 39 e0 87 40 f9 20 01 00 b4 57 f4 ff 97 f6 03
[   41.476416] skb tailroom: 000005a0: 00 aa a1 00 00 f0 21 94 3c 91 e6 dc ff 97 20 0c
[   41.484078] skb tailroom: 000005b0: 00 35 20 00 80 52 60 ba 04 b9 00 00 80 12 60 d2
[   41.491740] skb tailroom: 000005c0: 04 b9 e0 73 40 f9 80 01 00 b4 f0 f9 ff 97 02 02
[   41.499402] skb tailroom: 000005d0: 80 52 01 00 80 d2 67 df ff 97 61 32 53 39 02 08
[   41.507064] skb tailroom: 000005e0: 80 52 42 00 01 4b 01 00 80 12 21 20 c2 1a 21 00
[   41.514727] skb tailroom: 000005f0: 00 0a 61 d2 04 b9 e4 7f 40 f9 24 0d 00 b5 e0 8f
[   41.522389] skb tailroom: 00000600: 40 f9 60 12 00 b5 f4 77 40 f9 34 02 00 b4 e0 03
[   41.530051] skb tailroom: 00000610: 14 aa 39 f4 ff 97 61 c2 12 91 00 3c 00 94 1f 1c
[   41.537713] skb tailroom: 00000620: 00 72 61 01 00 54 85 01 00 f0 e0 03 14 aa 32 f4
[   41.545375] skb tailroom: 00000630: ff 97 e4 03 00 aa a0 84 43 f9 a1 00 00 f0 a2 42
[   41.553037] skb tailroom: 00000640: 2a 91 21 c0 36 91 43 72 80 52 f2 dd ff 97 f4 7b
[   41.560699] skb tailroom: 00000650: 40 f9 34 02 00 b4 e0 03 14 aa 27 f4 ff 97 61 d2
[   41.568363] skb tailroom: 00000660: 12 91 ee 3b 00 94 1f 1c 00 72 61 01 00 54 85 01
[   41.576028] skb tailroom: 00000670: 00 f0 e0 03 14 aa 20 f4 ff 97 e4 03 00 aa a0 84
[   41.583695] skb tailroom: 00000680: 43 f9 a1 00 00 f0 a2 42 2a 91 21 c0 36 91 e3 72
[   41.591355] skb tailroom: 00000690: 80 52 e0 dd ff 97 e0 83 40 f9 00 0e 00 b4 fc f5
[   41.599019] skb tailroom: 000006a0: ff 97 00 1c 00 12 00 00 00 52 60 ce 06 39 60 36
[   41.606678] skb tailroom: 000006b0: 42 39 60 3a 02 39 e0 43 40 f9 7f 3a 00 f9 a0 00
[   41.614330] skb tailroom: 000006c0: 00 b4 b2 f9 ff 97 17 dd ff 97 60 3a 00 f9 7f 36
[   41.621983] skb tailroom: 000006d0: 02 39 e0 47 40 f9 7f 3e 00 f9 40 0c 00 b4 ab f9
[   41.629643] skb tailroom: 000006e0: ff 97 10 dd ff 97 60 3e 00 f9 60 00 00 14 e1 03
[   41.637308] skb tailroom: 000006f0: 17 aa
[   41.641325] CPU: 0 PID: 0 Comm: swapper/0 Tainted: G         C        5.15.137 #0
[   41.648816] Hardware name: Raspberry Pi 4 Model B Rev 1.1 (DT)
[   41.654649] Call trace:
[   41.657091]  dump_backtrace+0x0/0x180
[   41.660772]  show_stack+0x18/0x30
[   41.664091]  dump_stack_lvl+0x6c/0x88
[   41.667761]  dump_stack+0x18/0x34
[   41.671080]  netdev_rx_csum_fault+0x70/0x74
[   41.675266]  __skb_checksum_complete+0xdc/0xe0
[   41.679714]  icmpv6_rcv+0xdc/0x4bc
[   41.683117]  ip6_protocol_deliver_rcu+0xd8/0x400
[   41.687737]  ip6_input+0x90/0xb0
[   41.690966]  ip6_mc_input+0x54/0xe0
[   41.694455]  ipv6_rcv+0xe8/0x130
[   41.697684]  __netif_receive_skb_one_core+0x48/0x60
[   41.702566]  netif_receive_skb+0x58/0x11c
[   41.706579]  br_pass_frame_up+0x130/0x1a0
[   41.710589]  br_handle_frame_finish+0x3bc/0x4f0
[   41.715119]  br_handle_frame+0x384/0x3d0
[   41.719041]  __netif_receive_skb_core.constprop.0+0x23c/0xcb4
[   41.724792]  __netif_receive_skb_one_core+0x2c/0x60
[   41.729674]  process_backlog+0xdc/0x250
[   41.733513]  __napi_poll+0x5c/0x1b4
[   41.737004]  net_rx_action+0x2dc/0x334
[   41.740756]  _stext+0x10c/0x28c
[   41.743897]  irq_exit+0xbc/0x100
[   41.747128]  handle_domain_irq+0x64/0x90
[   41.751053]  gic_handle_irq+0x74/0xa0
[   41.754723]  call_on_irq_stack+0x20/0x54
[   41.758647]  do_interrupt_handler+0x54/0x60
[   41.762832]  el1_interrupt+0x30/0x50
[   41.766410]  el1h_64_irq_handler+0x18/0x2c
[   41.770509]  el1h_64_irq+0x78/0x7c
[   41.773910]  arch_cpu_idle+0x18/0x2c
[   41.777487]  default_idle_call+0x20/0x70
[   41.781410]  do_idle+0x214/0x274
[   41.784639]  cpu_startup_entry+0x28/0x60
[   41.788562]  rest_init+0xc8/0xe0
[   41.791792]  arch_call_rest_init+0x10/0x1c
[   41.795892]  start_kernel+0x5d0/0x5f4
[   41.799555]  __primary_switched+0xa0/0xa8
```

herbetom commented:
How is the Radio behaving? Is the AP transmitting on 2,4 or 5 GHz? Both at the same time shouldn't be possible? Is it cappable of OWE and the OWE TM?

T-X commented:
@herbetom with the current config the radio comes up with one usable AP interface tuned to 2.4GHz. And no mesh interface. I also don't think 2.4GHz + 5GHz simultaneously would be possible, as it's still just one radio.
Haven't checked OWE, but can try next time.

T-X commented:
Changelog v2:
* added changes to .github/filters.yml, as created via `make update-ci`, to fix some complaints from the CI

---

To Freifunk Halle (@mfuhrmeisterDM, @Markus-Be, @3dfxatwork?), as you seem to be using quite a few Pi 4s with Raspbian already: How are you using them exactly? Which versions do you have, are there some manifest aliases you'd like to have added? Anything else that you'd need to make use of them with Gluon?

I'd also file a separate PR afterwards to add at least kmod-usb-net-rtl8152 to bcm27xx and x86 by default. Any other USB ethernet drivers people would need?

neocturne commented on targets/bcm27xx-bcm2711:
```diff
@@ -0,0 +1,7 @@
+device('raspberrypi-4-model-b', 'rpi-4', {
+	manifest_aliases = {
+		'raspberry-pi-4-model-b-rev-1.1',
+	}
```
```suggestion
	},
```
I would expect a trailing comma here... can we somehow configure our linter to complain in such cases?

grische commented on targets/bcm27xx-bcm2711:
```diff
@@ -0,0 +1,7 @@
+
+device('raspberrypi-4-model-b', 'rpi-4', {
+	manifest_aliases = {
+		'raspberry-pi-4-model-b-rev-1.1',
```
We have a bunch more devices currently, in particular rev-1.4

grische replied:
Here are the variants we have:
https://github.com/freifunk-gluon/gluon/pull/3073/files#diff-354a477dd339a057cbdd0607732186aa6c57d53f65c954dc41da432c580ec225R5-R9

We do not run Gluon v2019 anymore though, so I don't think all of them are required for us at least

neocturne replied:
Image names for all RPi targets are currently broken https://github.com/freifunk-gluon/gluon/issues/3099 - we can drop all these aliases when that has been fixed.

neocturne commented on targets/bcm27xx-bcm2711:
```diff
@@ -0,0 +1,10 @@
+include 'bcm27xx.inc'
+
+device('raspberrypi-4-model-b', 'rpi-4', {
+	manifest_aliases = {
```
Let's add a comment here similar to the other RPi targets, so we know we can drop them when versions before 2023.2 are EOL. The commit message should mention that the aliases are added only for upgrades from unreleased Gluon builds.

T-X commented:
Changelog v4:
* rebased to current master branch (no conflicts)
* added comment to manifest_alias and commit message regarding why the aliases were added
* (partially) retested on a Pi 4 rev. 1.5:
  *  `lua -e 'print(require("platform_info").get_image_name())'` now shows `raspberrypi-4-model-b` instead of `raspberry-pi-4-model-b-rev-1.5` (as expected since 8956b9d7e97681e49ea56c6db0a877f00297afc7, also related to #3110)
  * `sysupgrade` and `firstboot` still work as expected
